### ISSUE TITLE
Compilation error on round().astype(int)

### ISSUE THREAD

**Issue by jeanlaroche** [serge-sans-paille/pythran#854]
The code below does not compile
```
import numpy as np

# pythran export testFunc(int,float[])
def testFunc(x,A):
    B = np.round(A * x).astype(int)       # <- Does not compile.
```

The error is: 

> /Users/jlaroche/temp/JeanPythran/pythran/boost/simd/arch/x86/sse2/scalar/function/inearbyint.hpp:31:14: error: use of undeclared identifier '_mm_cvtsd_si64'
      return _mm_cvtsd_si64(_mm_set_sd(a0));
NOTE: 
This is on a mac, on linux, the code compiles just fine :(

**Comment by serge-sans-paille** [comment]
looks like a boost simd issue... @jfalcou ?

**Comment by jeanlaroche** [comment]
I confirm this is now fixed on master @ b966fe6c6aba34474880e2a086373cf6ee91345e
